2025-08-220 16:23 o.h.e.jdbc.spi.SqlExceptionHelper-Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt
2025-08-220 16:23 o.h.e.jdbc.spi.SqlExceptionHelper-Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt
2025-08-220 16:23 o.s.o.j.LocalContainerEntityManagerFactoryBean-Failed to initialize JPA EntityManagerFactory: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution [Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt] [n/a]
2025-08-220 16:23 o.s.boot.SpringApplication-Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution [Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt] [n/a]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1826)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:607)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:207)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:970)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1361)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1350)
	at com.order.ordersystem.OrdersystemApplication.main(OrdersystemApplication.java:12)
Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution [Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt] [n/a]
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:431)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.afterPropertiesSet(AbstractEntityManagerFactoryBean.java:400)
	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.afterPropertiesSet(LocalContainerEntityManagerFactoryBean.java:366)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1873)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1822)
	... 15 common frames omitted
Caused by: org.hibernate.exception.JDBCConnectionException: Unable to open JDBC Connection for DDL execution [Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt] [n/a]
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:51)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:58)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:108)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:94)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.getIsolatedConnection(DdlTransactionIsolatorNonJtaImpl.java:74)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.getIsolatedConnection(DdlTransactionIsolatorNonJtaImpl.java:39)
	at org.hibernate.tool.schema.internal.exec.ImprovedExtractionContextImpl.getJdbcConnection(ImprovedExtractionContextImpl.java:63)
	at org.hibernate.tool.schema.extract.spi.ExtractionContext.getQueryResults(ExtractionContext.java:43)
	at org.hibernate.tool.schema.extract.internal.SequenceInformationExtractorMariaDBDatabaseImpl.extractMetadata(SequenceInformationExtractorMariaDBDatabaseImpl.java:46)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.initializeSequences(DatabaseInformationImpl.java:66)
	at org.hibernate.tool.schema.extract.internal.DatabaseInformationImpl.<init>(DatabaseInformationImpl.java:60)
	at org.hibernate.tool.schema.internal.Helper.buildDatabaseInformation(Helper.java:185)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:93)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:280)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:144)
	at java.base/java.util.HashMap.forEach(HashMap.java:1430)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:141)
	at org.hibernate.boot.internal.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:37)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:324)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:463)
	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1517)
	at org.springframework.orm.jpa.vendor.SpringHibernateJpaPersistenceProvider.createContainerEntityManagerFactory(SpringHibernateJpaPersistenceProvider.java:66)
	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.createNativeEntityManagerFactory(LocalContainerEntityManagerFactoryBean.java:390)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:419)
	... 19 common frames omitted
Caused by: java.sql.SQLNonTransientConnectionException: Socket fail to connect to address=(host=localhost)(port=3306)(type=primary). Connection refused: getsockopt
	at org.mariadb.jdbc.client.impl.ConnectionHelper.connectSocket(ConnectionHelper.java:122)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:129)
	at org.mariadb.jdbc.Driver.connect(Driver.java:68)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at org.mariadb.jdbc.Driver.connect(Driver.java:26)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:137)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:360)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:111)
	at org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl.getConnection(DatasourceConnectionProviderImpl.java:126)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:483)
	at org.hibernate.resource.transaction.backend.jdbc.internal.DdlTransactionIsolatorNonJtaImpl.getIsolatedConnection(DdlTransactionIsolatorNonJtaImpl.java:46)
	... 39 common frames omitted
Caused by: java.net.ConnectException: Connection refused: getsockopt
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:628)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:533)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:583)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:282)
	at java.base/java.net.Socket.connect(Socket.java:665)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.connectSocket(ConnectionHelper.java:116)
	... 53 common frames omitted
2025-08-220 16:43 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto): [Field error in object 'memberCreateDto' on field 'password': rejected value [123]; codes [Size.memberCreateDto.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password],2147483647,8]; default message [패스워드의 길이가 너무 짧습니다.]] 
2025-08-220 16:44 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto) with 4 errors: [Field error in object 'memberCreateDto' on field 'password': rejected value []; codes [NotEmpty.memberCreateDto.password,NotEmpty.password,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password]]; default message [password는 필수 입력 항목입니다.]] [Field error in object 'memberCreateDto' on field 'email': rejected value []; codes [NotEmpty.memberCreateDto.email,NotEmpty.email,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.email,email]; arguments []; default message [email]]; default message [email은 필수 입력 항목입니다.]] [Field error in object 'memberCreateDto' on field 'name': rejected value []; codes [NotEmpty.memberCreateDto.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.name,name]; arguments []; default message [name]]; default message [이름은 필수 입력 항목입니다.]] [Field error in object 'memberCreateDto' on field 'password': rejected value []; codes [Size.memberCreateDto.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password],2147483647,8]; default message [패스워드의 길이가 너무 짧습니다.]] 
2025-08-220 16:46 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto): [Field error in object 'memberCreateDto' on field 'password': rejected value [2222]; codes [Size.memberCreateDto.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password],2147483647,8]; default message [패스워드의 길이가 너무 짧습니다.]] 
2025-08-220 16:48 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto): [Field error in object 'memberCreateDto' on field 'password': rejected value [1234]; codes [Size.memberCreateDto.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password],2147483647,8]; default message [패스워드의 길이가 너무 짧습니다.]] 
2025-08-220 16:52 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto): [Field error in object 'memberCreateDto' on field 'password': rejected value [123123]; codes [Size.memberCreateDto.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.password,password]; arguments []; default message [password],2147483647,8]; default message [패스워드의 길이가 너무 짧습니다.]] 
2025-08-220 16:53 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 16:54 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 16:54 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 16:54 c.o.o.c.s.CommonExceptionHandler-이미 존재하는 이메일입니다.
2025-08-220 16:55 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 16:55 c.o.o.c.s.CommonExceptionHandler-Validation failed for argument [0] in public org.springframework.http.ResponseEntity<?> com.order.ordersystem.member.controller.MemberController.create(com.order.ordersystem.member.dto.MemberCreateDto): [Field error in object 'memberCreateDto' on field 'name': rejected value []; codes [NotEmpty.memberCreateDto.name,NotEmpty.name,NotEmpty.java.lang.String,NotEmpty]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [memberCreateDto.name,name]; arguments []; default message [name]]; default message [이름은 필수 입력 항목입니다.]] 
2025-08-220 17:27 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 17:28 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
2025-08-220 17:28 c.o.o.c.s.CommonExceptionHandler-이메일 또는 비밀번호가 일치하지 않습니다.
